### PR TITLE
Refresh AT&T onboarding page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Refresh AT&T onboarding page as per [copydoc](https://docs.google.com/document/d/1QG0zxy9u_oGhXJUBsm3mNQy0l5dLwWOlDrheB_-BPH8/edit)

## QA

- Visit https://ubuntu-com-12052.demos.haus/engage/unlisted/at&t-onboarding
- See that the page matches the copy doc, and looks good

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5997

### CODE REVIEW

**Comment by sowasred2012** [comment]
@spencerbygraves could you take a look at this please?

**Comment by lyubomir-popov** [comment]
hi @petesfrench there's a lot I woul;d want to improve from a design perspective, but I don't suppose that would be in scope. As a quick fix, here's a version I made by messing around in the browser to improve the most thorny problems https://pastebin.canonical.com/p/JnWcJFW37N/

ubuntu-com-12052 demos haus_engage_unlisted_at t-onboarding (1)

File: templates/engage/unlisted/at&t-onboarding.html [hunk 1]
```diff
@@ -23,232 +23,193 @@
...
 
 <section class="p-strip--light">
-  <div class="row">
-    <div class="col-7">
-      <h2 class="p-heading--3">AT&T Pricing</h2>
-      <p>The Ubuntu Pro private offer includes custom pricing. AT&T can add Ubuntu Pro using the custom SKU in Azure for only $0.004 (less than half a penny) per core per hour.</p>
+  <div class="u-fixed-width">
+    <h2>Why an Ubuntu Pro Private Offer for Azure?</h2>
+  </div>
+  <div class="row" style="padding-top: 60px; padding-bottom: 60px;">
```
Comment (minkyngkm) [inline templates/engage/unlisted/at&t-onboarding.html:82]: Maybe it's for design review but are paddings necessary here?
The space looks a bit wide between "Why an Ubuntu Pro Private Offer for Azure?" and "Security and SLA Support" 
image

File: templates/engage/unlisted/at&t-onboarding.html [hunk 2]
```diff
@@ -23,232 +23,193 @@
...
       <thead>
         <tr>
-          <th>Support Feature</th>
-          <th class="u-align--center">AT&T Private Offer Ubuntu Pro w/ Advanced Support</th>
-          <th class="u-align--center">AT&T Private Offer Ubuntu Advantage w/ Advanced Support</th>
+          <th>Severity Level</th>
+          <th>Effort Period</th>
+          <th>Response Method</th>
+          <th>Initial Response Time</th>
+          <th>Subsequent Response Time</th>
```
Comment (minkyngkm) [inline templates/engage/unlisted/at&t-onboarding.html:141]: This tittle is clipped on small screens
image

**Review by minkyngkm — APPROVED** [review]
Looking good @sowasred2012 
I just left two small comments. Feel free to update them please.

File: templates/engage/unlisted/at&t-onboarding.html (hunk 2 above)
Comment (sowasred2012) [reply templates/engage/unlisted/at&t-onboarding.html:141]: Good shout - I've updated with the mobile card pattern to make it more readable.

File: templates/engage/unlisted/at&t-onboarding.html (hunk 1 above)
Comment (lyubomir-popov) [reply templates/engage/unlisted/at&t-onboarding.html:82]: even if it is a design choice, inline px values are hardly the best way to do this. Can you please use a strip instead.

**Comment by lyubomir-popov** [comment]
@sowasred2012 
image
Can you please make this look like an h2 so all headings on the page are the same size?

**Comment by lyubomir-popov** [comment]
sorry @sowasred2012 can you please add the same amount of white space under the table - atm it's too close to the next paragraph

**Comment by sowasred2012** [comment]
@lyubomir-popov yep - that's done

**Comment by lyubomir-popov** [comment]
@sowasred2012 all good, thanks. I've added the +1

**Pull request merged**